packages/webservice/models/build.gradle.kts

```
@@ -20,6 +20,14 @@ java {
     }
 }
 
+tasks.create<Jar>("sourcesJar") {
```

Can we change this to the format from [app/build.gradle.kts](https://github.com/yahoo/navi/blob/master/packages/webservice/app/build.gradle.kts#L38)

I think
```suggestion
val sourcesJar by tasks. registering(Jar::class) {
```